semantic/src/site/globals/site.overrides

```diff
@@ -454,18 +454,20 @@ a:hover {
 }
 
 #content-area .top-container {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
```

Comment:
You actually don't need these two lines, as the parent class `#content-area` already has "display:flex" and "flex-direction: column:", so this is redundant here.

Reply:
Removed before merging :)